_posts/developers/apps/inrupt-tutorial/2020-11-06-00_getting-started.md

```diff
@@ -112,398 +108,393 @@ Node.js installation.
+                </dl>
+            </div>
+    
+            <div id="read" class="panel">
+                <div class="row">
+                    <label id="readlabel" for="webID"
+                    >3. Read back name (anyone's!) from their WebID:
+                    </label>
+                    <input
+                            type="url"
+                            id="webID"
+                            name="webID"
+                            size="50"
+                            value="...not logged in yet - but enter any WebID to read from its profile..."
+                    />
+                    <button name="btnRead" id="btnRead">Read Profile</button>
+                </div>
+                <dl class="display">
+                    <dt>Formatted Name (FN) read from Pod:&nbsp</dt>
+                    <strong><span id="labelFN">...not read yet...</span></strong>
+                </dl>
+            </div>
+        </body>
+    </html>
+    ```
 
-    * Create an `index.js` file with the following content:
-
-        ```javascript
-        import {
-          getSolidDataset,
-          getThing,
-          setThing,
-          getStringNoLocale,
-          setStringNoLocale,
-          saveSolidDatasetAt
-        } from "@inrupt/solid-client";
-        
-        import { Session } from "@inrupt/solid-client-authn-browser";
-        
-        import { VCARD } from "@inrupt/vocab-common-rdf";
-        
-        // If your Pod is *not* on `solidcommunity.net`, change this to your identity provider.
-        const IDENTITY_PROVIDER = "https://solidcommunity.net";
-        document.getElementById("identity_provider").innerHTML = `[<a target="_blank" href="${IDENTITY_PROVIDER}">${IDENTITY_PROVIDER}</a>]`;
-      
-        const NOT_ENTERED_WEBID = "...not logged in yet - but enter any WebID to read from its profile...";
-        
-        const session = new Session();
-        
-        const buttonLogin = document.querySelector("#btnLogin");
-        const buttonWrite = document.querySelector("#btnWrite");
-        const buttonRead = document.querySelector("#btnRead");
-      
-        // Click button if the user hits the 'Enter' key when entering name.
-        document.querySelector("#input_name").addEventListener("keyup", event => {
-          if(event.key === "Enter") {
-            buttonWrite.click();
-            event.preventDefault();
-          }
-        });
-        
-        // Click button if the user hits the 'Enter' key when entering WebID.
-        document.querySelector("#webID").addEventListener("keyup", event => {
-          if(event.key === "Enter") {
-            buttonRead.click();
-            event.preventDefault();
-          }
+1. Create an `index.js` file with the following content:   
+
+    > **NOTE:** If your Pod is not on `https://solidcommunity.net`, modify the
+      value of the `SOLID_IDENTITY_PROVIDER` variable accordingly.
```

Comment:
This is the blurb that was relocated here about changing the SOLID_IDENTITY_PROVIDER